R/get_predicted_bayesian.R

```diff
@@ -43,7 +43,7 @@ get_predicted.stanreg <- function(x,
   fun_args <- list(x,
                    newdata = args$data,
                    re.form = args$re.form,
-                   nsamples = iterations,
+                   ndraws = iterations,
```

Comment:
This fixes the warning, but I'm not sure wether it might break stuff on very old brms versions 🤔

Reply:
We can check for which argument is used in the method, ala:

```r
fun_args <- list(
  # arguments other than ndraws
)
if ("nsamples" %in% 
  names(formals(getS3method(
    "posterior_predict",
    class(x)
  )))
) {
  fun_args <- c(
    fun_args, nsamples = iterations
  )
} else {
  fun_args <- c(
    fun_args, ndraws = iterations
  )
}
```

Reply:
We can check for which argument is used in the method, ala:

```r
fun_args <- list(
  # arguments other than ndraws
)
if ("nsamples" %in% 
  names(formals(getS3method(
    "posterior_predict",
    class(x)
  )))
) {
  fun_args <- c(
    fun_args, nsamples = iterations
  )
} else {
  fun_args <- c(
    fun_args, ndraws = iterations
  )
}
```

Reply:
why not, but isn't it overkill?

Reply:
Will the code break at some point, or is it just a warning? If it would be incompatible for older brms/rstanarm versions to use `ndraws`, I would definitely use the suggested code from Brenton, else it's probably a "matter of taste"...